lib/vsc/utils/exceptions.py

```diff
@@ -82,7 +82,10 @@ def __init__(self, msg, *args, **kwargs):
         @param msg: exception message
         @param *args: list of formatting arguments for exception message
         @param logger: logger to use
+        @param include_location: include location where error was raised from
         """
+        include_location = kwargs.get('include_location', False)
```

Comment:
if you always want to set it for a certain subclass, why not use a class constant? is there a use case where you would to set if for individual exceptions?